src/Elasticsearch/Framework/ElasticsearchOutdatedIndexDetector.php

```diff
@@ -31,23 +32,37 @@ class ElasticsearchOutdatedIndexDetector
      */
     private $helper;
 
+    /**
+     * @var Connection
+     */
+    private $connection;
+
     public function __construct(
         Client $client,
         ElasticsearchRegistry $esRegistry,
         EntityRepositoryInterface $languageRepository,
-        ElasticsearchHelper $helper
+        ElasticsearchHelper $helper,
+        Connection $connection
     ) {
         $this->client = $client;
         $this->registry = $esRegistry;
         $this->languageRepository = $languageRepository;
         $this->helper = $helper;
+        $this->connection = $connection;
     }
 
     /**
      * @return string[]
      */
     public function get(): ?array
     {
+        $exportingIndices = array_map(
+            static function (array $col) {
```

Comment:
Would you mind using `array_column` here instead? I think that would be a bit more readable.